content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -41,17 +41,17 @@ and returning when you're ready. If you are, let's dive in!
 
 ## Intro to Solana Mobile
 
-Native mobile wallets hold your private keys and use them to sign and send
-transactions just like web extension wallets. However native mobile wallets use
-the
+Native mobile wallets hold your secret keys (previously referred to as private
+keys) and use them to sign and send transactions just like web extension
+wallets. However native mobile wallets use the
```

Comment:
```suggestion
Native mobile wallets hold your secret keys and use them to sign and send transactions just like web extension
wallets. However native mobile wallets use the
```

Reply:
Thanks for reading CONTRIBUTING. We don't want to imply 'private' is deprecated, as web3.js will use 'private' as that's what webcrypto uses.